package.json

```diff
@@ -143,6 +143,11 @@
           "default": true,
           "markdownDescription": "Enable snippet completions from Angular language server. Requires using TypeScript 4.3+ in the workspace."
         },
+        "angular.suggest.autoImports": {
+          "type": "boolean",
+          "default": true,
+          "markdownDescription": "Enable/disable auto import suggestions."
+        },
```

Comment:
I think the name of this option is a bit misleading? If I understand correctly, it doesn't enable/disable all auto imports, just the "global" completions. Can you update this description and option name?

Reply:
Yes, it confuses me too. But this description is from the typescript. So I just copied it.

https://github.com/microsoft/vscode/blob/998fb692680cc20d70a805df8fa59cb1443bc897/extensions/typescript-language-features/package.nls.json#L90
https://github.com/microsoft/vscode/blob/998fb692680cc20d70a805df8fa59cb1443bc897/extensions/typescript-language-features/src/languageFeatures/fileConfigurationManager.ts#L208

Reply:
😅 well we can do better than that unless I’m missing something. Maybe the option in typescript evolved over time and used to mean what it says. Or maybe it was always just totally wrong 🤷‍♂️